resources/schema.gql

```diff
@@ -686,8 +691,8 @@ type FlowConfiguration {
 }
 
 type FlowConfigurationBatching {
```

Comment:
Does this control ingest or transform batching, or both? Perhaps we should rename it to make this more explicit?

Reply:
We don't know how push ingest will look like. Anyway, the configuration tries to be abstract from flow details. It could be  a totally different meaning, i.e., number of re-verified records for a potential verification flow.

Reply:
Or it could be the compactiion control - one thing is to compact by schedule, another thing is to compact when accumulated a million of records, but not less than once in a month.

Reply:
Decided to keep it as is for now. Will be more clear when adding new flow types.